projects/packages/publicize/src/enhanced-open-graph.php

```diff
@@ -0,0 +1,172 @@
+<?php
```

Comment:
We've discussed not including this in the package. Do you think we can remove it, or are there more dependencies to deal with first?

Reply:
Yup, I just added it temporarily to get publicize working, I am working on getting it removed.